recipes/default.rb

```diff
@@ -62,6 +62,12 @@
 end
 
 include_recipe 'java::default'
+if node['alfresco']['db_ssl_enabled'] == true
```

Comment:
This block must go after invoking artifact-deployer, otherwise the `/tmp/rds-combined-ca-bundle.pem` won't be there.

Also, the name of the pem file `/tmp/rds-combined-ca-bundle.pem` could be resolved with `node['artifacts']['ssl-db-creds']['destination']`